### ISSUE TITLE
User suggestion: adjustable amperage fuses

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/circuit-construction-kit-dc#22]
> Is there any way you can incorporate adjustable amperage fuses in this trainer?

**Comment by arouinfar** [comment]
@oliver-phet we haven't (yet) discussed the inclusion of fuses, but I can look into the possibility. Self-assigning for further investigation.

**Comment by ariel-phet** [comment]
I would say a fuse seems like a reasonable item to add, and should be fairly easy to program...it basically has two states, either more or less a no resistance wire, or a broken connection, and it would merely break as a result of current exceeding a particular value. Seems like it would be a valuable addition for people teaching circuits and the learning goals taught in basic physics classes.

**Comment by arouinfar** [comment]
Combining the requests for a fuse into one issue.

From #22
>I am a Professor at Conestoga College in Guelph, Ontario and currently teach Electrical/Electronics in our Truck and Coach programs. I wonder if you can tell me if you are looking at all at the following:
>1. Meter activities using a DMM
>2. Circuit simulations similar to a vehicle electrical system (ie starting circuit, lighting circuits (LED) etc)
>3. Adding more components to the simulator such as a ***fuse(very important)*** and perhaps a motor(DC,stepper etc) or something that can be customised for larger circuits
>4. ​Electronics components to simulate circuits such as 555 timer, regulators, diodes and rectifiers etc.

**Comment by arouinfar** [comment]
I agree @ariel-phet -- I think a fuse would be a worth-while addition.

**Comment by arouinfar** [comment]
A fuse was also requested in #3 
> I use Phet once in a while in my classroom and I was wondering if there is any way to add more components to the DC circuits. I am in automotive/Truck and Coach and would like different applications such as a ***fuse*** or circuit breaker, fans and different types of switches. Is this within the capabilities of the programmers or is this locked down to what we see is what we get?
On the plus side, great simulators for showing basic circuit faults.

**Comment by arouinfar** [comment]
Discussed in 3/9/17 design meeting.

Flow:
* If max current exceeded, fuse breaks (internal wire visibly broken)
* Increase current rating it becomes thicker
* If fuse is blown, reset button becomes available in the edit menu

In lifelike mode:
![image](https://cloud.githubusercontent.com/assets/8419308/23770016/9b8dc87a-04ce-11e7-8e50-cc3fdac88828.png)

In schematic mode:
![image](https://cloud.githubusercontent.com/assets/8419308/23769984/7b8019c0-04ce-11e7-8062-533257d1f777.png)

**Comment by arouinfar** [comment]
@samreid estimates 25 development hours + extra time for discussion, qa, etc.
